### PR TITLE
Add tests for CLI mode

### PR DESCRIPTION
**Pull request by kavitharaju**
Adds 20 testcases for CLI.

These tests are excluded when running on github actions. These tests require spawning child processes from node to run the terminal commands. This is running properly on local machine, but throwing error when run on github actions. Hence excluded them.

Also in this PR

- removes the output file option from CLI #98 
- report USFM errors properly in TSV and CSV conversions #97
- change the verseText creating code #99

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,9 +7,10 @@
         "test": "mocha --expose-gc --timeout 40000",
         "start": "node ../server.js",
         "lint": "eslint .",
-        "test-on-github": "mocha --expose-gc --timeout 40000 --ignore test/test_filesFromWild.js"
+        "test-on-github": "mocha --expose-gc --timeout 40000 --ignore test/test_filesFromWild.js --ignore test/test_cli.js"
     },
     "dependencies": {
+        "concat-stream": "^2.0.0",
```
Comment (joelthe1) [inline package.json:13]: Should this be a dev-dependency?

Comment (kavitharaju) [reply package.json:13]: yes. It is used in testing. So we can move it to dev-dependencies. Shall change that

Comment (joelthe1) [reply package.json]: Yes, let's do that.

Comment (kavitharaju) [reply package.json]: done

**Pull request merged**